pkg/storage/local.go

```diff
@@ -12,6 +12,11 @@ import (
 	"github.com/pingcap/errors"
 )
 
+const (
+	localDirPerm  = os.FileMode(0o755)
```

Comment:
Does hound mean this?
```suggestion
	localDirPerm  os.FileMode = 0755
```